source/hardware/controllers/native_instruments_traktor_kontrol_s4_mk3.rst

```diff
@@ -105,13 +113,13 @@ Here is how to select each mode:
 +================+===========================================================================+
 | Vinyl mode     | Press the "Jog" button (while the "Jog" button is off)                    |
 +----------------+---------------------------------------------------------------------------+
-| Jog mode       | Press the "Jog" button wgile in vinyl mode (while the "Jog" button is on) |
+| Jog mode       | Press the "Jog" button while in vinyl mode (while the "Jog" button is on) |
```

Comment:
no hwlabel for Jog / TT / Grid ?
just because they're round, not square like hwlabel? ; )

Reply:
That was exactly it yeah... :D 

I thought it could be misleading, but perhaps it is actually better anyway? I'll add it